dashboard/db/migrate/20210217182336_add_section_id_to_teacher_feedback.rb

```diff
@@ -0,0 +1,5 @@
+class AddSectionIdToTeacherFeedback < ActiveRecord::Migration[5.2]
+  def change
+    add_column :teacher_feedbacks, :section_id, :integer
+  end
```

Comment:
Might be worth adding an index on `section_id` if we'll be fetching all teacher feedbacks for a given section or otherwise looking up records by section and not by any existing index.